### ISSUE TITLE
New messages don't trigger `new-message` event

### ISSUE THREAD

**Issue by timgoeller** [cabal-club/cabal-client#78]
Haven't had the time yet to really check why this happens, but it seems to be caused by this commit: 1dbd5227923aa9063b93b57cfa9dbed31e246dda

If I join a cabal I will receive all historical messages, but not any of the new messages (or at least the event won't be triggered for new messages). This happens even when I join these channels after I've joined the cabal.

**Comment by cblgh** [comment]
thanks for the issue, tim! I'll have to dive back into this again at some point it seems :)

the commit linked was basically trying to figure out an issue in cabal-desktop, where messages would get duplicated now and then or sth like that

**Comment by cblgh** [comment]
@timgoeller where are you seeing the issues btw, in one of your cabal-bot repos?

**Comment by timgoeller** [comment]
Yeah, exactly, I've tried to upgrade the bridge/cabal-bot-core to the latest client version. I'll try to do some more testing in the next days.

**Comment by cblgh** [comment]
lmk how i can help @timgoeller! it'd be a great way for me to ease into dev again tbh :~
